src/components/Tabs/Annotation/AnnotationInputPopover.tsx

```diff
@@ -94,41 +97,46 @@ export default function AnnotationInputPopover(props: AnnotationInputPopoverProp
       const value = annotationData.getValueFromId(currentLabelIdx, lastClickedId) ?? "";
       setInputValue(value);
       originalValueRef.current = value;
+      const labelData = annotationData.getLabels()[currentLabelIdx ?? -1];
+      originalOptionsRef.current = Array.from(labelData.valueToIds.keys()).map((value) => ({ value, label: value }));
     }
     lastLabelIdxRef.current = currentLabelIdx;
     lastLabelCountRef.current = annotationData.getLabels().length;
   }, [activeEditRange]);
 
   // Focus the input when the popover is visible.
   useEffect(() => {
-    if (hasValidData && inputRef.current) {
+    if (hasValidData && inputContainerRef.current) {
       // Adding a slight delay fixes a bug where the input is not consistently
       // focused
       setTimeout(() => {
-        inputRef.current?.focus();
-        inputRef.current?.select();
+        const inputElement = inputContainerRef.current!.querySelector("input");
```

Comment:
Ant's AutoComplete does not directly give access to the Input ref, so we have to query for it instead.